assembl/static/widget/creativity/app/services.js

```diff
@@ -13,6 +13,55 @@ creativityServices.service('AssemblToolsService', ['$window', '$rootScope', '$lo
   };
 }]);
 
+creativityServices.service('LangStringService', ['$window', '$rootScope', '$log', function($window, $rootScope, $log) {
+  this.isMachineTranslation = function(langString) {
+    return langString["@language"].indexOf("-x-mtfrom-") > 0;
+  };
+
+  // returns the LangStringEntry (of this `langString` LangString) that is the most adapted to language `lang`
+  this.bestForLang = function(langString, lang){
```

Comment:
Is this logic duplicate of `best` in `models/langstring`?

Reply:
yes except that lang is only one language, not a whole ordered set of languages